samples/javascript_nodejs/85.bot-authentication-sni/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "bot-authentication-sni",
+    "version": "1.0.0",
+    "description": "Bot Builder v4 bot authentication with subject name/issuer sample",
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "index.js",
+    "scripts": {
+        "start": "node ./index.js",
+        "watch": "nodemon ./index.js",
+        "lint": "eslint .",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "@azure/identity": "^3.3.0",
+        "@azure/keyvault-certificates": "^4.7.0",
+        "@azure/keyvault-secrets": "^4.7.0",
+        "botbuilder": "~4.20.0",
```

Comment:
Should we target version 4.21.0? It's not published yet, but version 4.20 doesn't contain the MSAL support this sample needs.

Reply:
Or maybe we should use the `^` symbol.